src/api/middleware.py

```diff
@@ -1,14 +1,17 @@
-import json
 import time
-from datetime import datetime
+from django.utils import timezone
 
 from django.conf import settings
 from django.http import HttpResponse
 from django.utils.deprecation import MiddlewareMixin
 
+from tools.logger import DatabaseLogger
 from .utils.auth0utils import decode
 
 
+metrics_logger = DatabaseLogger(chunk_size=10)
```

Comment:
Might have asked this before -- I'd have expected 100 to be a better choice, but I don't know.